docs/README.md

```diff
@@ -0,0 +1,47 @@
+# Testing github pages locally
+
+This includes steps to test local changes related to github pages.
+
+Note: Current repo github pages leverages  `jekyll-theme-minimal` theme.
+
+* Install dependencies
+
+```bash
```

Comment:
Specify the distro+version this is applicable to. Also, if these tools are available in Azure Linux, it would be to provide steps for that as well.